config.inc.php

```diff
@@ -692,6 +692,8 @@ function x_send_sms_clickatell($to, $message) {
 //More details in README.password_expiration
 $CONF['password_expiration'] = 'YES';
 
+$CONF['version'] = '3.3-dev';
```

Comment:
Moving `$version` to config.inc.php doesn't look like a good idea to me. It's part of the code, not the configuration, so it shouldn't be configurable ;-) and besides that, we'll run into problems if someone still doesn't use a config.local.php.

Reply:
It's part of 'config.inc.php' which contains the default config  ...vs the local override (config.local.php) was kind of my thinking.